src/main/resources/application.yml

```diff
@@ -103,6 +103,5 @@ dpr:
     caseloads:
       host: ${api.base.url.manage.users}
       path: /users/me/caseloads
-  endpoint:
-    api:
-      roles: INCIDENT_REPORTS__RO,INCIDENT_REPORTS__RW,INCIDENT_REPORTS__APPROVE,INCIDENT_REPORTS__PECS
+    user:
+      role: INCIDENT_REPORTS__RO
```

Comment:
Framework only supports 1 role.  Will work with DPR to address this as we really want this protected by a "standard" system role

Reply:
😞